ferrocene/doc/user-manual/src/install.rst

```diff
@@ -6,52 +6,93 @@ Installing Ferrocene
 
 This chapter describes how to install and validate the Ferrocene toolchain.
 
+Before proceeding, you should identify the :doc:`targets <targets/index>` you
+want to install. You must pick the target of the host platform you're going to
+install Ferrocene on, and optionally one or more cross-compilation targets
+(depending on the hardware you'll deploy the executable or library on).
+
 Prerequisites
 -------------
 
-Ferrocene has platform-specific prerequisites, that vary based on the
-compilation targets you want to use. Follow the "Prerequisites" section of the
-target page for each of the targets you plan to use.
+The prerequisites of Ferrocene depend on the targets you are going to install.
+For each of the targets you identified above, follow the "prerequisites"
+section of the target documentation.
 
 Installation
 ------------
 
-Ferrocene is shipped as multiple tarballs, and all of them need to be
-installed. Refer to the list provided to you after the sale for the set of
-tarballs you need to download and install.
+.. note::
+
+   We are developing a tool similar to ``rustup``, able to manage your
+   Ferrocene installations and install Ferrocene without relying on a web
+   browser. It will be available soon.
+
+Manual installation
+~~~~~~~~~~~~~~~~~~~
+
+Ferrocene installation archives are available for manual download at
+`releases.ferrocene.dev <https://releases.ferrocene.dev>`_. The website
```

Comment:
Is that underscore supposed to be there?

```rst
`releases.ferrocene.dev <https://releases.ferrocene.dev>`_. The website
                                                         ^
```

Reply:
Yes, that is the rst syntax for links.